internal/rqlite/batching_test.go

```diff
@@ -0,0 +1,104 @@
+package rqlite
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/pbinitiative/zenbpm/internal/appcontext"
+	"github.com/pbinitiative/zenbpm/internal/rqlite/sql"
+	"github.com/pbinitiative/zenbpm/pkg/ptr"
+)
+
+func Test_NormalExecution_shouldExecuteRightAway(t *testing.T) {
+	// given
```

Comment:
I would get rid of these given, when, then, assert comments they do not provide any value.
All err != nil are assertions as well and are not commented as such.